homeassistant/components/verisure/config_flow.py

```diff
@@ -0,0 +1,173 @@
+"""Config flow for Verisure integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from verisure import (
+    Error as VerisureError,
+    LoginError as VerisureLoginError,
+    ResponseError as VerisureResponseError,
+    Session as Verisure,
+)
+import voluptuous as vol
+
+from homeassistant.config_entries import (
+    CONN_CLASS_CLOUD_POLL,
+    ConfigEntry,
+    ConfigFlow,
+    OptionsFlow,
+)
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import callback
+
+from .const import (  # pylint:disable=unused-import
+    CONF_GIID,
+    CONF_LOCK_CODE_DIGITS,
+    CONF_LOCK_DEFAULT_CODE,
+    DEFAULT_LOCK_CODE_DIGITS,
+    DOMAIN,
+    LOGGER,
+)
+
+
+class VerisureConfigFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Verisure."""
+
+    VERSION = 1
+    CONNECTION_CLASS = CONN_CLASS_CLOUD_POLL
+
+    installations: dict[str, str]
+    email: str
+    password: str
+
+    # These can be removed after YAML import has been removed.
+    giid: str | None = None
+    settings: dict[str, int | str] = {}
```

Comment:
Don't set a class attribute to a mutable object. That's error prone and rarely needed. If we want to default to empty dict, define the attribute in the instance init method.